src/mpid/ch4/src/ch4r_proc.c

```diff
@@ -48,7 +48,8 @@ int MPIDIU_build_nodemap(int myrank, MPIR_Comm * comm, int sz, int *out_nodemap,
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDIU_BUILD_NODEMAP);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDIU_BUILD_NODEMAP);
 
-    ret = MPIR_NODEMAP_build_nodemap(sz, myrank, out_nodemap, sz_out);
+    /* The nodemap is built in MPIR_pmi_init. Runtime rebuilding node_map not supported */
+    MPIR_Assert(0);
```

Comment:
Maybe remove this whole function?

Reply:
It is called by `MPIDIU_build_nodemap_avtid`. It appears that the original design is to have this function rebuild nodemap. The whole dynamic process system probably will need an overhaul at some point, so rather than removing these place holder function, I am tempted to leave it for later.